package-lock.json

```diff
@@ -0,0 +1,13870 @@
+{
```

Comment:
we're using yarn now, not npm, so you'll need to update yarn.lock instead

Reply:
Switched to yarn